rails_api_validation_errors.gemspec

```diff
@@ -16,11 +16,13 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
-  s.add_dependency "rails", "~> 4.0", ">= 4.0.0"
+  s.add_dependency "rails", "~> 5.2.0"
```

Comment:
Is `5.2.x` the real dependency here, or does it make sense to loosen it a bit (like `~> 5.0`)?

Reply:
5.0 works too 😄